templates/participants.yaml

```diff
@@ -0,0 +1,29 @@
+params:
+  name:
+    required: false
+    default: null
+  currency:
+    required: true
+  limit:
+    required: false
+    default: 100
```

Comment:
This might be something of a surprising default. @bushjames how many participants for our current project?

Reply:
this template config exists for example/documentation purposes and won't be used in prod.